grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/gradle/dependencies/MavenInstallArtifactsCalculator.kt

```diff
@@ -70,11 +67,7 @@ constructor(
     }
 
     fun calculate(): Map<String, List<MavenExternalArtifact>> {
-        val variantConfigs = calculateVariantConfigurations()
-        // Resolve the dependencies in each variant bucket from the configuration
-        val variantDependencies = resolveVariantDependencies(variantConfigs)
-        // Remove all dependencies from flavors which are already present in default.
-        return filterDependencies(variantDependencies)
+        return emptyMap()
```

Comment:
Are we deprecating this? perhaps we can use the `@Deprecated` annotation?

Reply:
Yes correct, added to whole class.